### PR TITLE
docs: comments-modal migration component review

### PR DESCRIPTION
**Pull request by b0ink**
# Description

Added a component review markdown for `./src/app/common/modals/comments-modal/comments-modal.coffee`

## Type of change

- [x] Documentation (update or new)

### CODE REVIEW

**Comment by atharv02-git** [comment]
# Ontrack Component-Comments-Modal Migration review #519 

## Review by Team Member Atharv

## Component Name: Comments Modal

## Component Review:
Hi Eliya, firstly you've done a great job migrating the "comments-modal" component, so Kudos to that!!!
The code seems to be cleaner, and more maintainable.
Here are few suggestions from my side:
1. Error Handling:
You could consider adding an error hadling mechanism in the service method (comments-modal.service.ts)
For Example: You could add an error handling mechanism in the service method that opens the modal.
`try {
} catch (error){
console.log(error)
}`
2. Code Documentation: Add comments and documentation to the new TypeScript code. This will help other developers understand the code more easily and maintain it in the future.

To summarise, the migration appears well-executed overall. With the above improvements, the component will be more robust, maintainable, and aligned with best practices. Therefore, after addressing the suggestions, the component should be good to go.

**Comment by satikaj** [comment]
This looks good. Good work!

**Review by satikaj — APPROVED** [review]
Looks good, thanks for fixing the CI issues.
